.github/workflows/housekeeping-lock-threads.yml

```diff
@@ -0,0 +1,44 @@
+name: "Housekeeping / Lock Threads"
+
+on:
+  schedule:
+    - cron: '*/15 * * * *'
```

Comment:
please use at least '0 * * * *'  or on: workflow_dispatch during debug to prevent resource leak